### DISCUSSION TITLE
Welcome to nginx_proxy Discussions!

### DISCUSSION THREAD

**Discussion by reiz** [reiz/nginx_proxy#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by malza321** [comment]
> 
> 
> ## 👋 Welcome!
> 
>   We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
>   * Ask questions you’re wondering about.
> 
>   * Share ideas.
> 
>   * Engage with other community members.
> 
>   * Welcome others and are open-minded. Remember that this is a community we
> 
>   build together 💪.
> 
> 
> 
>   To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> 
> 
> 
> 
> 
> 

Hi sir!

I want to make a my dns table like a hosts file and use it with forward proxy on nginx.

because I have no root permission on /etc/hosts and no dns server address as well. So I want to make that table and use it in nginx.conf.

it that possible?

plz help thanks.

**Reply by reiz** [reply]
I don't think that that's the right use case for the forward proxy. But I'm not an expert for DNS. Maybe somebody else can answer and help you.

**Comment by behoyh** [comment]
Hello, what's the license for this repo? I don't see any listed.

**Reply by reiz** [reply]
It's all MIT license! Do what ever you want with the code. But there is no warranty! 
I just added a license file to the repo and a note to the README.

**Comment by reiz** [comment]
MIT

Am Sa., 24. Apr. 2021 um 13:11 Uhr schrieb Beshoy Hanna <
***@***.***>:

> Hello, What's the license for this repo? I don't see any listed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/reiz/nginx_proxy/discussions/4#discussioncomment-653250>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAE7GYXNBYF5JJYVIZ2DQA3TKKRNFANCNFSM4VOJSJOA>
> .
>


-- 
*VersionEye GmbH*
Neckarauer-Waldweg 27
68199 Mannheim / Germany
Web: https://www.VersionEye.com <http://www.VersionEye.com>
Twitter: https://twitter.com/VersionEye
Amtsgericht: Mannheim
Handelsregister: Mannheim, HRB 720830
USt-IdNr.: DE282045007
CEO / Geschäftsführer: Robert Reiz

Die mit dieser e-Mail einschließlich ihrer Anlagen übermittelten
Informationen sind vertraulich. Sollten Sie diese e-Mail irrtümlich
erhalten haben, bitten wir Sie, uns hiervon zu informieren und die
übermittelten Informationen zu löschen.
Da die Gefahr besteht, dass unbefugte Dritte sich von den übermittelten
Daten Kenntnis verschaffen oder an ihnen manipulieren, können wir für den
Inhalt dieser e-Mail keine Haftung übernehmen. Wir übernehmen auch
keinerlei Haftung für Schäden, die durch etwaige Viren an Ihrem EDV-System
verursacht werden. Zur Risikobegrenzung empfehlen wir Ihnen, dieser E-Mail
angehängte Dateien durch Ihr eigenes Virus-Programm prüfen zu lassen.

**Comment by reiz** [comment]
I did not test version 1.19.10 yet. The Docker containers I provide are
tested with 1.18.
Not sure about the changes in 1.19.10. I recommend you check out the
changelogs and if you need further help you should contact the Nginx team
directly!


Am Mo., 3. Mai 2021 um 14:56 Uhr schrieb Junrae ***@***.***>:

> Hi
>
> I got update from 1.18 to 1.19.10
>
> and excuted with same config.
>
> there is no problem with 1.18 on proxy_pass but when I use 1.19.10, I got
> 403 forbidden from nginx.
>
> I don't know what is problem.
> Is there any different between 1.18 and 1.19?
>
> Everything is the same but only nginx binary file is different.
>
> Plz let me know any changes in 1.19.10.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/reiz/nginx_proxy/discussions/4#discussioncomment-687803>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAE7GYWK5D4C4YR35YBU3Z3TL2MQPANCNFSM4VOJSJOA>
> .
>


-- 
*VersionEye GmbH*
Neckarauer-Waldweg 27
68199 Mannheim / Germany
Web: https://www.VersionEye.com <http://www.VersionEye.com>
Twitter: https://twitter.com/VersionEye
Amtsgericht: Mannheim
Handelsregister: Mannheim, HRB 720830
USt-IdNr.: DE282045007
CEO / Geschäftsführer: Robert Reiz

Die mit dieser e-Mail einschließlich ihrer Anlagen übermittelten
Informationen sind vertraulich. Sollten Sie diese e-Mail irrtümlich
erhalten haben, bitten wir Sie, uns hiervon zu informieren und die
übermittelten Informationen zu löschen.
Da die Gefahr besteht, dass unbefugte Dritte sich von den übermittelten
Daten Kenntnis verschaffen oder an ihnen manipulieren, können wir für den
Inhalt dieser e-Mail keine Haftung übernehmen. Wir übernehmen auch
keinerlei Haftung für Schäden, die durch etwaige Viren an Ihrem EDV-System
verursacht werden. Zur Risikobegrenzung empfehlen wir Ihnen, dieser E-Mail
angehängte Dateien durch Ihr eigenes Virus-Programm prüfen zu lassen.
